ATM.py

```diff
@@ -0,0 +1,106 @@
+"""
+
+The script simulates ATM that preforms the following:
+balance check, withdraw cash, deposit cash and
+password change.
+
+"""
+import os
+import sys
+
+
+PATH=1
+BALANCE=1
+PASSWORD=0
+
+
+def file_to_dic(filename):
+    """
+
+    :arg
+        filename(.txt file) - parameter
```

Comment:
along with all your documentation sections, change the `-` sign with `:`.
and don't forget to add space between the attribute name and its type.
for example:
```suggestion
        filename (.txt file): parameter
```